third_party/packages/flutter_svg/lib/src/utilities/file.dart

```diff
@@ -1 +1 @@
-export '_file_io.dart' if (dart.library.html) '_file_none.dart';
+export '_file_io.dart' if (dart.library.js_interop) '_file_none.dart';
```

Comment:
Sadly, this might still rank this package as NOT wasm-compatible in pub due to issues with configuration specific imports.

Better to do
```suggestion
export '_file_none.dart' if (dart.library.io) '_file_io.dart';
```

Reply:
No I can confirm this works because I did it with the fork I published, flutter_svg_plus

Reply:
I have no doubt that it works at runtime. I'm concerned that the wasm classification on `pub.dev` will still be broken without this change.

Reply:
See https://github.com/dart-lang/pana/issues/1324

Reply:
See https://pub.dev/api/packages/flutter_svg/metrics

Search for package:flutter_svg/src/utilities/file.dart

Reply:
If you think the score for wasm on pub.dev is wrong, I assure you it is not.
See here https://pub.dev/api/packages/flutter_svg_plus/metrics and here https://pub.dev/api/packages/vector_graphics_compiler/metrics

Reply:
> See [dart-lang/pana#1324](https://github.com/dart-lang/pana/issues/1324)

I'm confused; I don't see anything in that issue that says that a conditional import based on `js_interop` wouldn't work, and it's currently working in practice. Could you point to a specific discussion/comment that makes you think this won't work in the future?